Framework/Algorithms/test/GetAllEiTest.h

```diff
@@ -543,7 +544,7 @@ class GetAllEiTest : public CxxTest::TestSuite {
     t1 = t_chop * l_mon2 / l_chop;
     t2 = (t_chop + Period) * l_mon2 / l_chop;
     {
-      auto &y = ws->dataY(1);
+      auto &y = ws->mutableY(1);
       for (size_t i = 0; i < y.size(); i++) {
         double t = 0.5 * (x[i] + x[i + 1]);
```

Comment:
I need some further help on this one, changing the for loops to `ws->points()` makes the test `void test_getAllEi()` fail. All of the other tests pass with the change to `ws->points()`.